sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_events_v1/query.sql

```diff
@@ -0,0 +1,1471 @@
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN tab_tray_private_mode_switched > 0
+          THEN client_id
+      END
+    ) AS tab_tray_private_mode_switched_users
+--app_icon_private_tab_tapped
+    ,
+    SUM(app_icon_private_tab_tapped) AS app_icon_private_tab_tapped,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN app_icon_private_tab_tapped > 0
+          THEN client_id
+      END
+    ) AS app_icon_private_tab_tapped_users
+--tab_tray_private_mode_tapped
+    ,
+    SUM(tab_tray_private_mode_tapped) AS tab_tray_private_mode_tapped,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN tab_tray_private_mode_tapped > 0
+          THEN client_id
+      END
+    ) AS tab_tray_private_mode_tapped_users
+--etp_setting_changed
+    ,
+    SUM(etp_setting_changed) AS etp_setting_changed,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN etp_setting_changed > 0
+          THEN client_id
+      END
+    ) AS etp_setting_changed_users
+--etp_settings
+    ,
+    SUM(etp_settings) AS etp_settings,
+    COUNT(DISTINCT CASE WHEN etp_settings > 0 THEN client_id END) AS etp_settings_users
+--etp_shield
+    ,
+    SUM(etp_shield) AS etp_shield,
+    COUNT(DISTINCT CASE WHEN etp_shield > 0 THEN client_id END) AS etp_shield_users
+--etp_tracker_list
+    ,
+    SUM(etp_tracker_list) AS etp_tracker_list,
+    COUNT(DISTINCT CASE WHEN etp_tracker_list > 0 THEN client_id END) AS etp_tracker_list_users
+/*Default browser*/
+--default_browser_changed
+    ,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN default_browser_changed > 0
+          THEN client_id
+      END
+    ) AS default_browser_changed_users,
+    COALESCE(SUM(default_browser_changed), 0) AS default_browser_changed
+/*Notification*/
+--re_engagement_notif_shown
+    ,
+    SUM(re_engagement_notif_shown) AS re_engagement_notif_shown,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN re_engagement_notif_shown > 0
+          THEN client_id
+      END
+    ) AS re_engagement_notif_shown_users
+--re_engagement_notif_tapped
+    ,
+    SUM(re_engagement_notif_tapped) AS re_engagement_notif_tapped,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN re_engagement_notif_tapped > 0
+          THEN client_id
+      END
+    ) AS re_engagement_notif_tapped_users
+/*Customize Home*/
+--app_menu_customize_homepage
+    ,
+    SUM(app_menu_customize_homepage) AS app_menu_customize_homepage,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN app_menu_customize_homepage > 0
+          THEN client_id
+      END
+    ) AS app_menu_customize_homepage_users
+--home_page_customize_home_clicked
+    ,
+    SUM(home_page_customize_home_clicked) AS home_page_customize_home_clicked,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN home_page_customize_home_clicked > 0
+          THEN client_id
+      END
+    ) AS home_page_customize_home_clicked_users
+  FROM
+    product_features
+  WHERE
+    submission_date >= start_date
+  GROUP BY
+    1
```

Comment:
As per our agreed SQL Style, we prefer using explicit field names in the GROUP BY section of the query (with specific exceptions to this rule):
https://docs.telemetry.mozilla.org/concepts/sql_style.html#be-explicit
```suggestion
    submission_date
```

This applied to all the query files.